pages/candy-machines/[candy_machine_ID].tsx

```diff
@@ -62,7 +63,6 @@ const CandyMachine: NextPage = () => {
 
     useEffect(() => {
         setError('')
-        fetchNfts()
```

Comment:
why this update? was it wrong before?

Reply:
Well yes, it was calling the function twice. I tested with 1 call and seems to work well!